python/BUILD

```diff
@@ -64,7 +64,7 @@ sphinx_to_adoc(
 
 test_to_example(
     name = "example_core",
-    input = "//python/tests/integration/core:test_core_example.py",
+    input = "//python/tests/integration/core:test_example.py",
     output = "python/core_example",
```

Comment:
Why no `.py`?

Reply:
Ideally, this file should not be exposed in the repo. Instead, it should be hidden in the sandbox, and the example is expected to be presented in the `README`. So I didn't want to make an impression like it's a real file to look at, more like a temporary configuration stuff.

We can the approach and make it a feature, though.